1209/AAAA/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: STM32 USB Serial Oscilloscope
+owner: Omega Centauri Software

```

Comment:
This needs to match the name of the organisation folder.

Reply:
1. There is no PCB design. If all one wants is to monitor TTL stuff, it should eventually work with a standard blue or black pill STM32F103 board with an F303 soldered on it.

2. Do you mean that I should remove the spaces from the owner name, to match the OmegaCentauriSoftware folder?

Reply:
Yes, the 'owner' field needs to be the directory name of your organisation.

Reply:
OK, fixed.